kola/tests/coretest/core.go

```diff
@@ -337,3 +338,12 @@ func TestUseradd() error {
 
 	return nil
 }
+
+// Test that /etc/machine-id isn't empty or COREOS_BLANK_MACHINE_ID
+func TestMachineID() error {
+	id := MachineID()
+	if id == "" || id == "COREOS_BLANK_MACHINE_ID" {
+		return fmt.Errorf("machine-id is %s", id)
```

Comment:
I'd split this into two if's becuase if it is an empty string this will be a very confusing error.